Magic/Spells/SummonSpell.cs

```diff
@@ -33,4 +35,20 @@ public override void Cast(Agent caster)
 
         summonerComponent.Summon(caster, GetAimedPosition(caster), data);
     }
+
+    public override bool IsAICastValid(Agent caster)
+    {
+        var summonerComponent = caster.GetComponent<SummonerAgentComponent>();
+        if (summonerComponent == null)
+        {
+            return true;
+        }
+
+        else if (summonerComponent.HasAnyActiveSummons() && Mission.Current.CurrentTime - summonerComponent.LastSummonTime > SPAWNED_TIME_LEFT_TO_DISMISS)
```

Comment:
An extra `else` we could remove here
```suggestion
        if (summonerComponent.HasAnyActiveSummons() && Mission.Current.CurrentTime - summonerComponent.LastSummonTime > SPAWNED_TIME_LEFT_TO_DISMISS)
```